lib/repl.js

```diff
@@ -806,7 +806,8 @@ REPLServer.prototype.complete = function(line, callback) {
           });
         }
       } else {
-        this.eval(expr, this.context, 'repl', function(e, obj) {
+        const evalExpr = `try { ${expr} } catch (e) { undefined }`;
```

Comment:
You’re right. If you want it removed, sure, but I don’t mind the verbosity of writing the “return” value down explicitly.